Main/ClientApp/src/Api/ConfigurationApi.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  authorizedGet,
+  authorizedPatch,
+} from "../Infrastructure/FetchUtilities";
+
+class ConfigurationApi {
+  async getConfiguration(): Promise<Map<string, string>> {
```

Comment:
I don't like this approach. The way it is done now, you use the controller to give access from the react code to the database - it should not be done like this. All the business login and code should be done in the backend. So rather than passing all of the configuration table content, backend should create a proper configuration object and pass it to here. This way when we have a class, it also acts as a contract of what can be done.

Reply:
good point